backend/mobilityAI/Controllers/PatientsController.cs

```diff
@@ -163,6 +170,71 @@ public IActionResult GetPatientActivity(int patientId, long start, long end)
             return Ok();
         }
 
+        /// <summary>
+        /// Adding patient achievements. If current achievement stats exist, update the information. 
+        /// If it doesn't exist, add a new row to the database for that patient
+        /// </summary>
+        /// <param name="patientId">
+        /// The id value of the patient
+        /// </param>
+        /// <param name="steps">
+        /// The number of steps goal that the patient wishes to achieve 
+        /// </param>
+        /// <param name="activityTime">
+        /// The number of steps goal that the patient wishes to achieve
+        /// </param>
+        /// <returns>
+        /// 200 if successful
+        /// </returns>
+        [HttpPost("PatientAchievements")]
+        public IActionResult PatientAchievements(int patientId, int steps, int activityTime)
```

Comment:
We already store the patient's walking baseline mobility in the database, see:
https://github.com/pgunasekara/mobilityai/blob/414eeea264e427c394e320632142507a4200de8e/frontend/mobilityai_react/App/Screens/NewPatientForm.js#L135

We should add another field in the `NewPatientForm.js` for the steps goal and store it alongside the rest of the json.

Also, we can clean up the endpoint a little by getting the patientId from the url like this:

```C#
[HttpPost("{patientId}/Achievements")]
public IActionResult PatientAchievements(int patientId, int steps, int activityTime)
```